src/molecules/full-story-content/full-story-content.tsx

```diff
@@ -9,8 +10,13 @@ import { StoryPageSlots } from "../slots";
 const { DefaultStoryCardSlot } = StoryPageSlots;
 
 // Renders all the cards in the story
-export const FullStoryContent = ({ story }) => {
+export const FullStoryContent = ({ story, config }) => {
   return story.cards.map((card, cardIdx) => {
+    const customCardRender = get(config, ["opts", "render", "storyCardRenderProps"], {});
```

Comment:
rename to `storyCardRender` since we're following the same convention elsewhere in amplib